package.json

```
@@ -11,7 +11,8 @@
     "fmt": "prettier --write .",
     "lint": "eslint .",
     "prepare": "husky install",
-    "precommit": "lint-staged"
+    "precommit": "lint-staged",
+    "build": "lerna run build"
```

We're using npm workspaces here, we can utilise that to build all packages instead of lerna:
```suggestion
    "build": "npm run --workspaces build"
```